crates/ui/src/components/scrollbar.rs

```diff
@@ -9,33 +9,27 @@ use gpui::{
     UniformListScrollHandle, View, WindowContext,
 };
 
-pub struct Scrollbar {
+pub struct Scrollbar<T: ScrollableHandle> {
     thumb: Range<f32>,
-    state: ScrollbarState,
+    state: ScrollbarState<T>,
     kind: ScrollbarAxis,
 }
 
 /// Wrapper around scroll handles.
 #[derive(Clone, Debug)]
-pub enum ScrollableHandle {
+pub enum ListScrollableHandle {
```

Comment:
I guess we could do away with the `ListScrollableHandle` altogether now that this is traitified.

Reply:
Yes, make sense.